mllib/src/test/scala/org/apache/spark/mllib/classification/LogisticRegressionSuite.scala

```diff
@@ -394,6 +394,7 @@ class LogisticRegressionSuite extends SparkFunSuite with MLlibTestSparkContext w
     lrA.optimizer.setNumIterations(numIteration)
     val lrB = new LogisticRegressionWithLBFGS().setIntercept(true).setFeatureScaling(false)
     lrB.optimizer.setNumIterations(numIteration)
+    lrB.optimizer.setRegParam(0.001)
```

Comment:
If we do not penalize, it will not yield the same result in the scaled space due to poor convergence rate.